ClinGen/clincoded
# Connects to #1528. Connects to #1547. Connects to #1616. Connects to #1719.

**Steps to test #1528:**
1. Create a GDM under an affiliation, and start/score an evidence (e.g. segregation LOD score, Proband score).
2. Save the classification for the scored evidence and proceed with provisioning/approving the classification.
3. Return to the Gene-Disease Record page and expect to see your classification now in a new panel below the header.
![image](https://user-images.githubusercontent.com/6956310/46070021-b0e29e00-c131-11e8-9f00-77ff5c47608c.png)
3. Now switch to a different affiliation and access the same GDM created above.
4. On the Gene-Disease Record page, expect to see the previously approved classification under **Other classifications** header in the panel.
![image](https://user-images.githubusercontent.com/6956310/46071554-213eee80-c135-11e8-84e7-b27cd5e69cdf.png)
5. Click on the link icon next to the **APPROVED** label and expect to see the respective evidence summary being shown in a new window/tab.

**Steps to test #1547:**
1. Start a new interpretation in the VCI under an affiliation and proceed with provisioning/approving it.
2. Return to the interpretation view, and expect to see the interpretation classification and status in a new panel on the **Basic Info** tab.
![image](https://user-images.githubusercontent.com/6956310/46072296-d9b96200-c136-11e8-9181-e9156b913849.png)
3. Now switch to a different affiliation and access the VCI evidence view using the same variant.
4. On the **Basic Info** tab, expect to see the previously approved interpretation classification in the same panel.
![image](https://user-images.githubusercontent.com/6956310/46072676-d4a8e280-c137-11e8-8eee-e868529e02ff.png)
5. Click on the link icon next to the **APPROVED** label and expect to see the respective evaluation summary being shown in a new window/tab.

**Steps to test #1616:**
1. Start a new interpretation in the VCI and associate it with an **Other** or **Unknown** Mode of Inheritance (which does not have any adjectives).
2. Proceed with saving the summary and provisioning it.
3. Expect to see **absence** of _null_ instances in where the Mode of Inheritance is shown.
![image](https://user-images.githubusercontent.com/6956310/46073512-f4410a80-c139-11e8-9e7a-4805c73dff17.png)
![image](https://user-images.githubusercontent.com/6956310/46073611-45e99500-c13a-11e8-9ab0-5500504551fe.png)
4. Click on the **View Provisional Summary** button in the snapshot and expect to see the evaluation summary page being loaded in a new window/tab.
5. On the summary page, also expect to see the **absence** of _null_ instance in where the Mode of Inheritance is shown.
![image](https://user-images.githubusercontent.com/6956310/46073487-df647700-c139-11e8-81e4-380e861ced7c.png)

**Steps to test #1719:**
1. Create a new GDM under the **ClinGen Curator Test Affiliation 1** affiliation.
2. Start/score some evidence (e.g. segregation LOD score, Proband score, Case-Control).
3. Save the classification for the scored evidence and proceed with provisioning/approving the classification.
4. Subsequently, expect to be able to publish the approved classification to the Data Exchange staging server.
